modules/identity/src/Volo.Abp.Identity.Blazor/Pages/Identity/RoleManagement.razor.cs

```diff
@@ -28,6 +40,97 @@ public RoleManagement()
             ManagePermissionsPolicyName = IdentityPermissions.Roles.ManagePermissions;
         }
 
+        protected override ValueTask SetEntityActionsAsync()
+        {
+            UIExtensions.Instance
+                .EntityActions
+                .Get<RoleManagement>()
+                .AddIfNotContains(new EntityAction[]
```

Comment:
`AddIfNotContains` prevents duplicate adding. However, wouldn't it better to make this into the module's `ConfigureServices` method (with `OneTimeRunner` wrapper) to not execute it every time component is rendered.

Reply:
See my comment below.